static/js/util/ProfileTab.js

```diff
@@ -42,9 +44,12 @@ class ProfileTab extends React.Component {
     }));
     this.languageOptions = _.sortBy(languageOptions, 'label');
     this.privacyOptions = [
-      { value: 'public', label: 'Public to the world'},
-      { value: 'public_to_mm', label: 'Public to other micromasters students'},
-      { value: 'private', label: 'Private'}
+      { value: 'public', label: 'Public to the world', helper: 'We will publish your Micromasters ' +
+        'profile on our website.' },
+      { value: 'public_to_mm', label: 'Public to other micromasters students', helper: 'Your Micromasters profile ' +
+        'will only be viewable by other learners in your program, and by MIT faculity and staff.' },
+      { value: 'private', label: 'Private', helper: 'Your Micromasters profile will be viewable only by ' +
+        'MIT faculty and staff.' }
```

Comment:
can you use template literals here instead of string concatenation? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals